### DISCUSSION TITLE
Release 5.3.3

### DISCUSSION THREAD

**Discussion by miguel-a-calles-mba** [serverless-heaven/serverless-webpack#1546] [Team Posts]
Hi,

I created a 5.3.3 milestone.

Whenever you have time, may you please tag the PRs we should include in the next release?

Depending on the level of changes, we might need to consider the release as a minor update (i.e., 5.4.0) instead of a patch (i.e., 5.3.3).

Thanks!

**Comment by HyperBrain** [comment]
Remember that feature updates which are not backwards compatible must be
put into a minor version and breaking changes in a major update.
Only bugfixes or completely backwards compatible updates can be put into a
patch version. Otherwise you would break automated deployment systems which
allow installing patch updates without notice (and a rollback there if e.g.
a dependency update is declined in the deployment system).

**Comment by HyperBrain** [comment]
Just to add:

In my experience it worked well to have minor and patch milestones in
parallel and assign the PRs according to their impact into the milestones.
Then you can safely release sequentially.

**Comment by miguel-a-calles-mba** [comment]
Thanks for the clarification.

It seems like we need a 5.3.3 milestone for dependency updates and a 5.4.0 milestone for non-breaking PRs.

**Comment by miguel-a-calles-mba** [comment]
I created `release/5.3.3` and `release/5.4.0` branches to support parallel releases per @HyperBrain's suggestion. The PRs will merge into their respective branches.
